oci/oci_test.go

```diff
@@ -88,18 +88,23 @@ func TestAppendDevicePermissionsFromCgroupRules(t *testing.T) {
 		},
 		{
 			doc:         "major out of range",
-			rule:        "c 18446744073709551616:1 rwm",
-			expectedErr: `invalid major value in device cgroup rule format: 'c 18446744073709551616:1 rwm'`,
+			rule:        "c 2048:1 rwm",
```

Comment:
See my other comment; we'd have to check if 2048 is indeed out of range (it may not be).

Possibly we may have to skip range validation (other than conversion to the `LinuxDeviceCgroup.Major` / `LinuxDeviceCgroup.Minor`), and leave it to the OCI runtime and the Kernel itself to decide wether or not the value is "ok"

Reply:
Although I think range validation is important (especially because it takes place on `create` now and the Kernel will only check on `start`, which could give a user false confidence in their rules), the general uncertainty surrounding it, with both the `runtime-spec` and `runc` ignoring the limit, gives me pause. Maybe we just return a warning, something along the lines of  `this value probably will be out of range`.